tests/test_cli_socket_autodiscovery.py

```
@@ -63,35 +66,54 @@ def _run(self) -> None:
+
+    def _handle_connection(self, conn: socket.socket) -> None:
+        with conn:
+            conn.settimeout(2.0)
+            data = b""
+            try:
+                while b"\n" not in data:
+                    chunk = conn.recv(4096)
+                    if not chunk:
+                        break
+                    data += chunk
+            except (ConnectionResetError, socket.timeout):
```

Ruff (0.15.13)

[warning] Replace aliased errors with `TimeoutError`

Replace with builtin `TimeoutError`

(UP041)